docs/20251227_2354_その15政治活動費の費目別シート分割設計.md

```
@@ -0,0 +1,156 @@
+
+| 項目名 | タグ名 | 説明 |
+|--------|--------|------|
+| 費目 | HIMOKU | 費目の名称（例：会議費、交通費） |
+| 合計 | KINGAKU_GK | そのシート内の合計金額 |
+| その他の支出 | SONOTA_GK | 5万円以下の支出の合算 |
+
+## 設計
+
+### データフロー
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)